plenum/test/txn_author_agreement/acceptance/test_taa_acceptance_validation.py

```diff
@@ -288,8 +276,27 @@ def test_taa_acceptance_not_allowed_when_disabled(
     with pytest.raises(
         validation_error,
         match=(
-            r"Txn Author Agreement acceptance is disabled"
-            " and not allowed in requests"
+            r"Incorrect Txn Author Agreement"
         )
     ):
         validate_taa_acceptance(request_dict)
+
+
+
+#@pytest.mark.skip()
+def test_taa_acceptance_retired(
+        tconf, txnPoolNodeSet, validate_taa_acceptance, validation_error,
+        turn_off_freshness_state_update, max_last_accepted_pre_prepare_time,
+        request_dict, latest_taa, monkeypatch,
+        looper, sdk_pool_handle, sdk_wallet_trustee, set_txn_author_agreement
+):
+    validate_taa_acceptance(request_dict)
+    taa_data = set_txn_author_agreement(latest_taa.text, latest_taa.version, retired=1)
```

Comment:
If this test passes then it seems our implementation doesn't match requirements )